tally_ho/apps/tally/views/reports/turnout_reports_by_admin_areas.py

```diff
@@ -168,28 +175,30 @@ def get_initial_queryset(self):
                 race=F('center__resultform__ballot__electrol_race_id')
                 ).values('id').annotate(
                 races=ArrayAgg('race', distinct=True),
-                number=F('station_number'),
-                num_registrants=F('registrants')
                 )
             voters = 0
             stations_processed = 0
             registrants_in_processed_stations = 0
             for station in station_ids_by_race:
+                station_obj = stations_by_id.get(station.get('id'))
```

Comment:
Same Could raise `AttributeError` if `station.get('id')` returns None

Reply:
The above response also applies here